app/build.gradle

```diff
@@ -48,7 +48,6 @@ android {
             minifyEnabled true
             shrinkResources true
             debuggable false
-            applicationIdSuffix ".android.playmarket"
```

Comment:
@guristas That was added in #27 for appropriate funnel attribution. Is it intentional or occasional, that you've removed this?

Reply:
No, I deleted it meaningfully.